renderdoc/driver/vulkan/wrappers/vk_resource_funcs.cpp

```diff
@@ -2477,7 +2512,18 @@ VkResult WrappedVulkan::vkCreateImage(VkDevice device, const VkImageCreateInfo *
            next->sType == VK_STRUCTURE_TYPE_EXTERNAL_FORMAT_ANDROID)
         {
           isExternal = true;
-          break;
+
+          // we can't call vkGetImageMemoryRequirements on AHB-backed images until they are bound
+          if(next->sType == VK_STRUCTURE_TYPE_EXTERNAL_MEMORY_IMAGE_CREATE_INFO)
+          {
+            VkExternalMemoryImageCreateInfo *extCreateInfo =
+                (VkExternalMemoryImageCreateInfo *)FindNextStruct(
+                    &createInfo_adjusted, VK_STRUCTURE_TYPE_EXTERNAL_MEMORY_IMAGE_CREATE_INFO);
+            resInfo.imageInfo.isAHB =
+                (extCreateInfo->handleTypes &
+                 VK_EXTERNAL_MEMORY_HANDLE_TYPE_ANDROID_HARDWARE_BUFFER_BIT_ANDROID) > 0;
```

Comment:
Style nit: using `> 0` to check a bitfield is totally valid but seems a little odd, I'd prefer `!= 0`.